develop-docs/development-infrastructure/devservices.mdx

```diff
@@ -76,3 +76,42 @@ Common modes:
 ```shell
 devservices up --mode symbolicator
 ```
+
+## Migrating data from `sentry devservices`
```

Comment:
What are your thoughts on phrasing it like "Migrating data from the deprecated `sentry devservices`"?

Reply:
sounds good to me